sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_pii_entities_async.py

```diff
@@ -0,0 +1,567 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import pytest
+import platform
+import functools
+
+from azure.core.exceptions import HttpResponseError, ClientAuthenticationError
+from azure.core.credentials import AzureKeyCredential
+from asynctestcase import AsyncTextAnalyticsTest
+from testcase import GlobalTextAnalyticsAccountPreparer
+from testcase import TextAnalyticsClientPreparer as _TextAnalyticsClientPreparer
+from azure.ai.textanalytics.aio import TextAnalyticsClient
+from azure.ai.textanalytics import (
+    TextDocumentInput,
+    VERSION,
+    ApiVersion
+)
+
+# pre-apply the client_cls positional argument so it needn't be explicitly passed below
+# the first one
+TextAnalyticsClientPreparer = functools.partial(_TextAnalyticsClientPreparer, TextAnalyticsClient, api_version=ApiVersion.V3_1_preview_1)
+# TODO: add back offset and length checks throughout this test once I add them
+
+class TestRecognizePIIEntities(AsyncTextAnalyticsTest):
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    async def test_no_single_input(self, client):
+        with self.assertRaises(TypeError):
+            response = await client.recognize_pii_entities("hello world")
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    async def test_all_successful_passing_dict(self, client):
+
+        docs = [{"id": "1", "text": "My SSN is 555-55-5555."},
+                {"id": "2", "text": "Your ABA number - 111000025 - is the first 9 digits in the lower left hand corner of your personal check."},
+                {"id": "3", "text": "Is 998.214.865-68 your Brazilian CPF number?"}]
+
+        response = await client.recognize_pii_entities(docs, show_stats=True)
+        self.assertEqual(response[0].entities[0].text, "555-55-5555")
+        self.assertEqual(response[0].entities[0].category, "U.S. Social Security Number (SSN)")
+        self.assertEqual(response[1].entities[0].text, "111000025")
+        # self.assertEqual(response[1].entities[0].category, "ABA Routing Number")  # Service is currently returning PhoneNumber here
```

Comment:
curious if this is fixed now

Reply:
it's not :'(

Reply:
ah :( it's being used in the sample, I'm thinking we should remove it then.

Reply:
Just ran the samples and it turns out it creates two entries for this one entity. One is a phone number, the other is an ABA Routing number. I'll remove it from the samples for now